addOns/sqliplugin/src/main/java/org/zaproxy/zap/extension/sqliplugin/SQLiPayloadManager.java

```diff
@@ -79,6 +79,7 @@
     private static final String PAYLOAD_FILE = "resources/payloads.xml";
     private static final String TAG_BOUNDARY = "boundary";
     private static final String TAG_TEST = "test";
+    private static final Random RAND = new Random();
```

Comment:
This will introduce contention, though guess not a real problem here.

Reply:
If there are multiple scans and they happen to hit use of the random at the same moment?

Reply:
That's very unlikely to happen (in any case it shouldn't be that much time, compared to I/O, rtt).

Reply:
Okay